services/bower/bower-version.tester.js

```diff
@@ -44,10 +49,15 @@ t.create('Version label should be `no releases` if no stable version')
   .expectBadge({ label: 'bower', message: 'no releases' })
 
 t.create('Version label should be `no releases` if no pre-release')
-  .get('/vpre/bootstrap.json')
+  .get('/v/bootstrap.json?include_prereleases')
   .intercept(nock =>
     nock('https://libraries.io')
       .get('/api/bower/bootstrap')
       .reply(200, { normalized_licenses: [], latest_release_number: null })
   )
   .expectBadge({ label: 'bower', message: 'no releases' })
+
+t.create('Version (legacy redirect: vpre)')
+  .get('/vpre/bootstrap.json', { followRedirect: false })
+  .expectStatus(301)
```

Comment:
Badges are verified by `.expectBadge()` assertion. How about adding a new custom assertion for redirects?
```diff
diff --git a/core/service-test-runner/icedfrisby-shields.js b/core/service-test-runner/icedfrisby-shields.js
index ec1748da..4b1070f9 100644
--- a/core/service-test-runner/icedfrisby-shields.js
+++ b/core/service-test-runner/icedfrisby-shields.js
@@ -53,0 +54,4 @@ const factory = superclass =>
+    expectRedirect(location) {
+      return this.expectStatus(301).expectHeader('Location', location)
+    }
+
```
and
```diff
diff --git a/services/bower/bower-version.tester.js b/services/bower/bower-version.tester.js
index 613c4900..58502377 100644
--- a/services/bower/bower-version.tester.js
+++ b/services/bower/bower-version.tester.js
@@ -62,2 +62 @@ t.create('Version (legacy redirect: vpre)')
-  .expectStatus(301)
-  .expectHeader('Location', '/bower/v/bootstrap.json?include_prereleases')
+  .expectRedirect('/bower/v/bootstrap.json?include_prereleases')
```

Reply:
Again, agreed. I've added that helper here and changed it for the services already touched in this PR. I'll leave updating the other service tests to use it for a different PR (maybe several) to avoid scope creep. There are about 30 other services to update.